package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "spellcheck": "mdspell --en-gb --report --ignore-acronyms --ignore-numbers --no-suggestions 'content/**/*.md'",
     "broken-link-local": "blcl --recursive --ordered --exclude-external ./public/",
-    "broken-link-all": "blcl --recursive --ordered ./public/",
+    "broken-link-all": "blcl --recursive --exclude farnell --ordered ./public/",
```

Comment:
Why are we excluding farnell here?

Reply:
Farnell links are excluded as they cause the link checker to hang indefinitely. See earlier build https://circleci.com/gh/sourcebots/docs/535 It is not entirely clear why this is the case. One theory was that the User Agent for the link checker was being filtered, but this is not the case.

Reply:
Please could we have an inline comment to that effect?

Reply:
Meta (and well beyond the scope of this PR): this is a class of error which I see as unavoidable and a reason not to have external link checking.

Reply:
Inline comment added: de78cf8